src/lib/config.js

```diff
@@ -1,20 +1,6 @@
 'use strict'
 
 const Config = require('five-bells-shared').Config
```

Comment:
I kept the convention of uppercase because all the modules in shared are uppercase, including Utils